### DISCUSSION TITLE
Hello Friends, I need help with some Map related code.

### DISCUSSION THREAD

**Discussion by ursvr** [ORelio/Minecraft-Console-Client#2757] [Q&A]
My English is bad and I use a translator. The essence of the problem is that, according to an algorithm, I receive cards from the server to pass the captcha. But in Spain several cards are used for this. A more precise 3x3 grid, in which each element is part of the captcha. As a result, I receive 9 messages with parts of the map. I would like to combine them into one grid. I tried to analyze the source code, but a solution could not be found. The great GPT gives the code, but where to put it idk

This is what I get:
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/79226865/18f8e9cb-9f16-43df-b069-0b92f6f48c4d)
This is what I would like to receive:
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/79226865/5b40796b-395c-474d-9290-297db53bff42)

**Comment by milutinke** [comment]
We do not support automatic captcha bypassing, neither we will implement such a feature.

**Reply by ursvr** [reply]
> We do not support automatic captcha bypassing, neither we will implement such a feature.

I know. I do not need it. I want it to be convenient to enter captcha.

UPD: The algorithm is as follows: The console receives Maps - saves them - combines them into 1 photo - sends them to discord - I enter the captcha manually
